### ISSUE TITLE
RDA Issue: Use of the "Uniform Resource Locator" element

### ISSUE THREAD

**Issue by tmqdeborah** [RDARegistry/rballs#14]
The RDA Toolkit definition for the "Uniform Resource Locator" element is: "A Uniform Resource Locator, or URL, is the address of a remote access resource"

This element is in Chapter 4 which provides elements that are "used to obtain or access a resource"

I interpret this to mean that when the element is used for Manifestation data, it is intended to be used only to provide access to the actual resource that is described by the Manifestation data. 

If a URL is for information about various aspects of the resource, then it should be provided as appropriate for those aspects, e.g.:
- A URL for Biographical information about a person associated with a resource, should be provided as data for the "Biographical Information" element for the Person. 
- A URL for Sample text information could be provided as Note on Expression  data, preceded by "Sample text:"

Question: Am I correct in interpreting the definition this way?

**Comment by GordonDunsire** [comment]
I do not think this is a correct interpretation. It confuses URLs with URIs. A URL is not an identifier for a thing, it is a location for a document.

In FRBR terms, the document (e.g. an entry in an online biographical dictionary) is a separate Work. I do not think RDA intends attribute-type data to be treated as relationship-type data; it offers separate elements in order to accommodate the distinction. This is not the same situation as, say, for the element Place of publication and the entity Place; it is a feature of RDA that it provides rich relationships between WEMI entities, and is relatively less focussed on relationships between WEMI entities and other RDA entities, or between other RDA entities.

RDA 4.6.1.3 says "Record the Uniform Resource Locator for the online resource being described.", so the use case involves using the URL for the Manifestation being described (although technically, it is an Item that the URL locates).

RDA goes on to say "Record a Uniform Resource Locator for a related resource as part of the description of the related manifestation." The key word is "part"; RDA does not intend the URL to be the whole data in the description (structured or unstructured), and preliminary boilerplate text such as "Sample text" should not be considered as part of the description of the related resource.

I do not think it would be appropriate to merely record a citation as the data for Biographical information, whether it is a URL or a textual citation. Using RDA's Alice Munro example, I don't think "https://en.wikipedia.org/wiki/Alice_Munro" as the complete content of the element is acceptable; neither is "For biographical information, see https://en.wikipedia.org/wiki/Alice_Munro." Rather, this would be a Subject relationship between the Wikipedia article and the Person named Alice Munro.
